src/styles/index.scss

```diff
@@ -1,3 +1,84 @@
-body {
-  margin: 0;
+.body {
```

Comment:
Class names should be kebab-case according to the code style rules. Rename '.body' to '.body-container' or another appropriate name that follows these conventions.